test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -3283,6 +3283,10 @@ public string F2(int a)
     }
 }");
                 var msbuildPath = @"C:\Program Files (x86)\MSBuild\14.0\Bin";
+                if (!Directory.Exists(msbuildPath))
```

Comment:
Consider adding a test utility method for getting the MSBuild path to avoid duplicate code.